adagrams/game.py

```diff
@@ -1,11 +1,150 @@
+import random
 def draw_letters():
-    pass
+   letter_pool = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+     }
+   #taking an empty list to store drawn letters
+   drawn_letters =[]
+# using random.choice
+#    while len(drawn_letters) < 10:
+
+#     #extracting all the keys from dictionary
+#     letter_pool_keys = letter_pool.keys()
+#     letter = random.choice(list(letter_pool_keys))
+#     if letter_pool[letter] > 0:
+#         drawn_letters.append(letter)
+
+#         #adding random letter to the list and decreasing the frequency of the letter
+#         letter_pool[letter]-= 1
+#    return drawn_letter
```

Comment:
Remove unused code so that it doesn't clutter up your project or confuse other devs reading your code. Having unused, commented out code can also introduce bugs if this code is accidentally executed.

If you'd like to show me an alternate solution for some code you wrote, you can add that in the pull request as a comment.